assets/js/script.js

```diff
@@ -2,182 +2,174 @@
+    } else if (task.status === "in-progress") {
+      inProgressList.append(createTaskCard(task));
+    } else if (task.status === "done") {
+      doneList.append(createTaskCard(task));
     }
-
-    $('.draggable').draggable({
-        opacity: 0.7,
-        zIndex: 100,
-        helper: function (e) {
-            const original = $(e.target).hasClass('ui-draggable')
-                ? $(e.target)
-                : $(e.target).closest('.ui-draggable');
-            return original.clone().css({
-                width: original.outerWidth(),
-            });
-        },
-    });
+  }
+
+  $(".draggable").draggable({
+    opacity: 0.7,
+    zIndex: 100,
+    helper: function (e) {
+      const original = $(e.target).hasClass("ui-draggable")
+        ? $(e.target)
+        : $(e.target).closest(".ui-draggable");
+      return original.clone().css({
+        width: original.outerWidth(),
+      });
+    },
+  });
 }
 
 // Todo: create a function to handle adding a new task
-function handleAddTask(event){
-    event.preventDefault();
-
-    const taskName = $('#Task-Name').val();
-    const taskDescription = $('#Task-Description').val();
-    const taskDueDate = $('#Task-Due').val();
-
-    const newTask = {
-        id: generateTaskId(),
-        name: taskName,
-        description: taskDescription,
-        dueDate: taskDueDate,
-        status: 'to-do',
-    };
-    
-    const tasks = readTasksFromStorage();
-    tasks.push(newTask);
-
-    saveTasksToStorage(tasks);
-    renderTaskList();
-    
-    $('#Task-Name').val('');
-    $('#Task-Description').val('');
-    $('#Task-Due').val('');
+function handleAddTask(event) {
+  event.preventDefault();
+
+  const taskName = $("#Task-Name").val();
+  const taskDescription = $("#Task-Description").val();
+  const taskDueDate = $("#Task-Due").val();
+
+  const newTask = {
+    id: generateTaskId(),
+    name: taskName,
+    description: taskDescription,
+    dueDate: taskDueDate,
+    status: "to-do",
+  };
+
+  const tasks = readTasksFromStorage();
+  tasks.push(newTask);
+
+  saveTasksToStorage(tasks);
+  renderTaskList();
+
+  $("#Task-Name").val("");
+  $("#Task-Description").val("");
+  $("#Task-Due").val("");
 }
 
 // Todo: create a function to handle deleting a task
-function handleDeleteTask(event){
-    const taskId = $(this).data("task-id");
-    const tasks = readTasksFromStorage();
+function handleDeleteTask(event) {
+  const taskId = $(this).data("task-id");
+  const tasks = readTasksFromStorage();
 
-    tasks.forEach((task, index) => {
+  tasks.forEach((task, index) => {
     if (task.id === taskId) {
-        tasks.splice(index, 1);
+      tasks.splice(index, 1);
     }
-    });
+  });
 
-    saveTasksToStorage(tasks);
+  saveTasksToStorage(tasks);
 
-    renderTaskList();
+  renderTaskList();
 }
 
 // Todo: create a function to handle dropping a task into a new status lane
 function handleDrop(event, ui) {
+  const taskId = Number(ui.draggable[0].dataset.taskId);
```

Comment:
I converted the taskId here into a number so it would be the same type as tasks.id